binder/environment.yml

```diff
@@ -3,10 +3,9 @@ channels:
 dependencies:
 # We can not refer to an environment.yml file from another
 # So to get performant launches on mybinder.org, we have copied
-# the contents of this requirements/conda.txt to this file.yml.
+# the contents of this requirements/main.yml to this file.yml.
```

Comment:
Should this comment include the `linux.yml` file too?

Reply:
Fixed in my last commit.